part-configuration.tex

```diff
@@ -48,146 +48,12 @@ \section{PostgreSQL}
 \section{LedgerSMB}
 \label{sec-global-config-ledgersmb}
 
-\subsection{ledgersmb.conf}
-\label{subsec-global-config-ledgersmb-conf}
+LedgerSMB configuration using 'ledgersmb.conf' is deprecated as of 1 Jan 2023. New enhancements may only available when using `ledgersmb.yaml` configuration file.

```

Comment:
I propose "New functionality" over "New enhancements": my first reading of enhancements was that it referred to the configuration itself, whereas "new functionality" can hardly refer to the config and necessarily refers to the application as a whole.

Reply:
Agreed, working on it.

Reply:
Done